magda-search-api-node/src/createApiRouter.ts

```diff
@@ -0,0 +1,216 @@
+import * as express from "express";
+import * as _ from "lodash";
+import buildJwt from "@magda/typescript-common/dist/session/buildJwt";
```

Comment:
Probably `magda-typescript-common/src/dist/session/buildJwt` just to consistent with other project plus you don't have to build `typescript-common` first before run this